terasoluna-tourreservation-domain/src/main/java/org/terasoluna/tourreservation/domain/repository/age/AgeRepository.java

```diff
@@ -19,6 +19,6 @@
 
 public interface AgeRepository {
 
-    Age findOne(String ageCode);
+    Age findById(String ageCode);
```

Comment:
- [ ] Please apply `Optional` as JPA. (Adjust the repository signature for easy maintenance.)

See https://github.com/terasolunaorg/terasoluna-tourreservation/pull/583/commits/c9080fcd0cb3b4a8f28410d5e437d2424dd76a42

Reply:
Adaptation of Optional.